### PR TITLE
Change it to a more better dark themed color

### PR DESCRIPTION
**Pull request by ItsNotAlexy**
changed the dark colour to something more nice to look at :)

### CODE REVIEW

**Comment by toolifelesstocode** [comment]
Can you explain what this pull request is for, by editing the pull request description.

**Review by ooliver1 — CHANGES_REQUESTED** [review]
Wouldn't it make more sense for these to be variables so you only have to change it in one place

File: css/history.css [hunk 1]
```diff
@@ -1,3 +1,3 @@
 body {
-    background: #23272A;
+    background: #1B2430;
```
Comment (toolifelesstocode) [inline css/history.css:2]:
```suggestion
    background: #5c18ae;
```

File: css/404.css [hunk 2]
```diff
@@ -1,15 +1,15 @@
 body {
-    background-color: #23272A;
+    background-color: #1B2430;
```
Comment (toolifelesstocode) [inline css/404.css:2]:
```suggestion
    background-color: #5c18ae;
``` 
Maybe try this purple, the website looks fine from the preview, but just very dark.

File: css/404.css [hunk 3]
```diff
@@ -1,15 +1,15 @@
...
 }
 
 .desc {
     text-align: center;
-    color: #23272A;
+    color: #1B2430;
```
Comment (toolifelesstocode) [inline css/404.css:7]:
```suggestion
    color: #5c18ae;
```

File: css/home.css [hunk 4]
```diff
@@ -1,12 +1,12 @@
 body {
-    background: #23272A;
+    background: #1B2430;
```
Comment (toolifelesstocode) [inline css/home.css:2]:
```suggestion
    background: #5c18ae;
```

File: css/home.css [hunk 5]
```diff
@@ -1,12 +1,12 @@
...
 }
 .desc {
     text-align: center;
-    color: #23272A;
+    color: #1B2430;
```
Comment (toolifelesstocode) [inline css/home.css:6]:
```suggestion
    color: #5c18ae;
```

File: css/home.css [hunk 6]
```diff
@@ -32,4 +32,9 @@ a:hover {
     bottom: 0;
     padding: 15px 0;
     width: 100%;
+}
+
+.jumbotron {
+    background-color: #1B2430;
```
Comment (toolifelesstocode) [inline css/home.css:38]:
```suggestion
    background-color: #5c18ae;
```

**Review by toolifelesstocode — CHANGES_REQUESTED** [review]
Maybe make the purple not that dark, by using this hex color #5c18ae

**Comment by toolifelesstocode** [comment]
@ItsNotAlexy Bump on this.

**Comment by ItsNotAlexy** [comment]
Closing because of new rewrite.

**Pull request closed without merging**